pkg/metricsDashboard/logger.go

```diff
@@ -16,8 +16,9 @@ const nodeInfoEndpoint = "nodes"
 const nodeConnectionEndpoint = "uptimes"
 const dealsEndpoint = "deals"
 
+var host = os.Getenv("API_HOST")
```

Comment:
@walkah I moved this out of the function call, I don't think we need to call it every time the function executes right?

Reply:
You're absolutely right! Good catch 👍